src/context/ThemeContext.jsx

```diff
@@ -0,0 +1,42 @@
+import { createContext, useState, useMemo } from "react";
+import { createTheme } from "@mui/material/styles";
+
+export const themeSetup = (mode) => {
+  return {
+    palette: {
+      mode: mode,
+      ...(mode === "dark"
+        ? {
+            background: {
+              default: "#141b2d",
+              contrast: "#8D8A8A",
+            },
+          }
+        : {
+            background: {
+              default: "#3225E1",
+              contrast: "#EBEBEB",
+            },
+          }),
+    },
+  };
+};
+
+export const ThemeContext = createContext({ toggleColorMode: () => {} });
```

Comment:
Ideą contextu jest to że przez niego przesyłamy informacje dalej. Dwa pozostałe exporty nie są potrzebne.